doc/htmldoc/synapses/handling_connections.rst

```diff
@@ -1,7 +1,10 @@
 .. _handling_connections:
 
-How to handle connections (synapses)
-=====================================
+How to handle created connections
+=================================
```

Comment:
```suggestion
How to inspect and modify connections
==============================
```
Also in the text below, "already created" sounds strange. Either write "existing connections" or just "connections", because when you talk about inspecting and modifying, that implies that the connections must exist.